wc_langPacks/wc_langPack_EN.js

```diff
@@ -103,4 +103,4 @@ wc_addLanguagePack({
     }
     return ret;
   },
-});
+});
```

Comment:
It's good practice to always keep a trailing empty line.
```suggestion
});

```